en/admin/control.rst

```diff
@@ -2142,3 +2162,358 @@ The following shows how to use the CUBRID Manager (hereafter, CM) Administrator
         ::
 
             cm_admin changedbinfo -p 33000 testcm testdb
+
+.. _cubrid-javasp-server:
+
+CUBRID Java Stored Procedure Server
+===================================
+
+Starting CUBRID Java SP Server
+------------------------------
+
+The following example shows how to start CUBRID Java SP server for *demodb*.
+
+To start the Java SP server, the java_stored_procedure parameter in the CUBRID configuration file (cubrid.conf) must set to yes.
+
+::
+
+    % cubrid javasp start demodb
+
+    @ cubrid javasp start: demodb
+    ++ cubrid javasp start: success
+
+The following message is returned if CUBRID Java SP server is already running. 
+
+::
+
+    % cubrid javasp start demodb
+
+    @ cubrid javasp start: demodb
+    ++ cubrid javasp 'demodb' is running.
+
+For details on other types of errors that may occur when starting the server, see :ref:`cubrid-javasp-server-errors`.
+
+Stopping CUBRID Java SP Server
+------------------------------
+
+The following example shows how to stop CUBRID Java SP server for *demodb*. 
+
+::
+
+    % cubrid javasp stop demodb
+
+    @ cubrid javasp stop: demodb
+    ++ cubrid javasp stop: success
+
+The following message is returned when CUBRID Java SP server has stopped. 
+
```

Comment:
The following message is returned when CUBRID Java SP server has been already stopped.

Reply:
has been already stopped. --> has been stopped already.  
 might be more fluent.

Reply:
Thank you, I'll fix it